### PR TITLE
[zh-cn] sync blog: 2023-02-03-sig-instrumentation-spotlight.md

### PR DESCRIPTION
**Pull request by kevin1689-cloud**
Fix [#39770](https://github.com/kubernetes/website/issues/39770)

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-02-03-sig-instrumentation-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+**作者**: Imran Noor Mohamed (Delivery Hero)
+
+**译者**: [Kevin Yang](https://github.com/kevin1689-cloud)
+
+<!--
+Observability requires the right data at the right time for the right consumer
+(human or piece of software) to make the right decision. In the context of Kubernetes,
+having best practices for cluster observability across all Kubernetes components is crucial.
+-->
+
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-02-03-sig-instrumentation-spotlight.md:27]: The blank line at line 27 is not necessary. 

The blank line followed by the commented en text is not required. Same below.

Comment (kevin1689-cloud) [reply content/zh-cn/blog/_posts/2023-02-03-sig-instrumentation-spotlight.md]: @windsonsea Done. Please take a look. Thanks!

**Comment by kevin1689-cloud** [comment]
@windsonsea Hi, I noticed some checks were not successful, as I only removed the blank line and pushed the file again, I don't know the reason why these checks were failed. Could you help find the problem, and I will try my best to fix it.
image

File: content/zh-cn/blog/_posts/2023-02-03-sig-instrumentation-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+**作者**: Imran Noor Mohamed (Delivery Hero)
+
+**译者**: [Kevin Yang](https://github.com/kevin1689-cloud)
+
+<!--
+Observability requires the right data at the right time for the right consumer
+(human or piece of software) to make the right decision. In the context of Kubernetes,
+having best practices for cluster observability across all Kubernetes components is crucial.
+-->
+可观测性需要在正确的时间提供正确的数据，以便正确的消费者（人员或软件）做出正确的决策。
```
[the remark this answers is not in the document]
Comment (kevin1689-cloud) [reply content/zh-cn/blog/_posts/2023-02-03-sig-instrumentation-spotlight.md]: > ”正确“有点儿硬，可以考虑换成”合适“。

我保留了“正确的决策”，因为我觉得可观测性最终的结果是要做出正确的决策。其他几个“正确”改成了“合适”，修改后的句子如下：
可观测性需要在合适的时间提供合适的数据，以便合适的消费者（人员或软件）做出正确的决策。

**Comment by kevin1689-cloud** [comment]
@tengqm 已根据修改意见修改，麻烦看一下是否有问题，谢谢

**Comment by windsonsea** [comment]
> 

Thank. Just a friendly reminder that if you have made the changes based on the suggested comments, please click "Resolve conversation", so everyone can easily learn the PR progress.

/lgtm

**Comment by kevin1689-cloud** [comment]
> > 
> 
> Thanks. Just a friendly reminder that if you have made the changes based on the suggested comments, please click "Resolve conversation" to confirm, so everyone can easily learn the PR progress.
> 
> /lgtm

Got it, thanks for the remind, I will do the confirm in the future PR.

**Pull request merged**